watertap/examples/case_studies/municipal_treatment/municipal_treatment.py

```diff
@@ -208,47 +201,10 @@ def set_operating_conditions(m):
 
 
 def initialize_system(m):
-    unit_list = ['feed', 'intake_pump', 'coag_and_floc',
-                 'sedimentation', 'ozonation', 'gravity_basin',
-                 'gac', 'backwash_pump', 'uv', 'anion_exchange',
-                 'chlorination', 'storage', 'recharge_pump']
-    unit_inlet_arc_dic = {'feed': None,
-                          'intake_pump': 's01',
-                          'coag_and_floc': 's02',
-                          'sedimentation': 's03',
-                          'ozonation': 's04',
-                          'gravity_basin': 's05',
-                          'gac': 's06',
-                          'uv': 's07',
-                          'backwash_pump': 's08',
-                          'anion_exchange': 's09',
-                          'chlorination': 's10',
-                          'storage': 's11',
-                          'recharge_pump': 's12'}
-    unit_inlet_properties_dic = {'feed': 'properties',
-                          'intake_pump': 'properties',
-                          'coag_and_floc': 'properties',
-                          'sedimentation': 'properties_in',
-                          'ozonation': 'properties_in',
-                          'gravity_basin': 'properties_in',
-                          'gac': 'properties_in',
-                          'uv': 'properties_in',
-                          'backwash_pump': 'properties',
-                          'anion_exchange': 'properties_in',
-                          'chlorination': 'properties_in',
-                          'storage': 'properties',
-                          'recharge_pump': 'properties'}
-
-    for u_str in unit_list:
-        if unit_inlet_arc_dic[u_str] is not None:
-            unit = getattr(m.fs, u_str)
-            arc = getattr(m.fs, unit_inlet_arc_dic[u_str])
-            inlet_blk = getattr(unit, unit_inlet_properties_dic[u_str])
-            propagate_state(arc)
-            if unit_inlet_properties_dic[u_str] == 'properties_in':
-                flags = fix_state_vars(inlet_blk)
-                solve(unit)
-                revert_state_vars(inlet_blk, flags)
+    seq = SequentialDecomposition()
+    seq.options.tear_set = []
+    seq.options.iterLim = 1
+    seq.run(m, lambda u: u.initialize())
```

Comment:
Just curious- why did we need to include this? I'd think we wouldn't need to go this far for a flowsheet with zero-order models.

Reply:
I'm not sure. Tim had a much more complicated initialization routine which I replaced with this. I didn't try removing it altogether.

And this really isn't complicated ... we should probably write a utility that just does this. It just initializes everything in order (which is fine because this flowsheet is a DAG).